### PR TITLE
menubar fix in mobile view

### PR DESCRIPTION
**Pull request by dhavalsavalia**
Changes:
1. Added a changed Bootstrap stylesheet with "navbar-toggle" class changed to show background colour(#513278) same as PyCon India 2018 Theme
2. Added "navbar-inverse" class to "nav" tag
2. Added jQuery and Bootstrap JavaScript files

### CODE REVIEW

**Comment by dhavalsavalia** [comment]
Check screenshots of Google Chrome and Safari [here](https://imgur.com/a/jYuYRjy)

**Comment by ananyo2012** [comment]
@dhavalsavalia Please add the style changes to styles.css instead of the bootstrap stylesheet. Also keep using othr bootstrap cdn instead of locally adding it.

**Comment by dhavalsavalia** [comment]
@ananyo2012 I have retained the Bootstrap v3.3.2 CDN version along with it's JavaScript. I have also added Pipfile and Pipfile.lock

File: themes/inpycon2018/templates/base.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <meta name="author" content="Pycon India">
     <link rel="shortcut icon" href="favicon.png">
     <!-- Global CSS -->
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
```
Comment (ananyo2012) [inline themes/inpycon2018/templates/base.html]: Is it required to change the cdn path ? Doesn't it work with the original cdn ?

File: themes/inpycon2018/templates/base.html [hunk 2]
```diff
@@ -108,5 +108,7 @@
       </div>
     </div>
   </footer>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (ananyo2012) [inline themes/inpycon2018/templates/base.html]: Are the js dependencies required ? If not don't add them since more dependencies means more load time.

Comment (dhavalsavalia) [reply themes/inpycon2018/templates/base.html]: Sadly, Bootstrap's Javascript doesn't work without jQuery.

screen shot 2018-05-27 at 9 44 43 am

Comment (dhavalsavalia) [reply themes/inpycon2018/templates/base.html]: And navbar-toggle will not work without Bootstrap JavaScript.

**Comment by vipulgupta2048** [comment]
@dhavalsavalia Any updates on the review made on this PR?

**Comment by ananyo2012** [comment]
This is good to be merged. Any changes you see that is needed to be made @vipulgupta2048 ?

**Comment by vipulgupta2048** [comment]
I wanted to see the commits being squashed because of their overlapping nature. But I think we can oversee that.

**Comment by ananyo2012** [comment]
@vipulgupta2048 GitHub has this cool feature of Squash and merge directly from github web which makes life of a reviewer much easier. But still contributors should always follow the practice of squashing  commits.

**Comment by vipulgupta2048** [comment]
Yes, there is. I personally never tried it though. But I like to push for squashing just to implement good practice in new contributors. I am fine with it eitherwise as when Github merges, either if its one or 10 commits it gets made into a single commit. So doesn't matter much.

**Pull request merged**